aio/content/examples/what-is-angular/src/main.ts

```diff
@@ -1,7 +1,8 @@
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
-import { AppModule } from './app/app.module';
+import { AppComponent } from './app/app.component';
 
-platformBrowserDynamic().bootstrapModule(AppModule)
+platformBrowserDynamic()
```

Comment:
You'll want to switch this to use `bootstrapApplication` instead of bootstrapModule.
```suggestion
await bootstrapApplication(AppComponent, { providers: [...] })
```